Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,49 @@ label monika_why_do_you_read:
     extend 3eua "While my situation in here has its downsides, [line_mid] opportunity to spend more time on the things I like."
     m 1ekbsu "...Though then again, nothing could ever beat spending more time with you~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_tabletop_rping",
+            category=['monika','literature'],
+            prompt="Tabletop Roleplaying",
+            random=True,
+            aff_range=(mas_aff.NORMAL, None)
+        )
+    )
+
+label monika_tabletop_rping:
+    m "Say, [player], there's this literature genre that I've been meaning to bring up lately."
```

Comment:
I know the brings it up as 'a mix between a game and a form of literature' later, but calling it straight up a literature genre feels like a bit of a stretch to me.
`"Say, [player], there's something I've been meaning to bring up lately."`?

Reply:
Dungeons and Dragons is for sure a game, but the genre as a whole is very "literary" in nature more than anything, particularly simple deviations of 'pretend' or 'make believe', where you just have the same concept but with chance elements. The core of TRPGs after all is the story element, which is the core in most literary works.